plugin/evm/config.go

```diff
@@ -205,6 +206,10 @@ type Config struct {
 	// This is useful for validators that don't need to index transactions.
 	// TxLookupLimit can be still used to control unindexing old transactions.
 	SkipTxIndexing bool `json:"skip-tx-indexing"`
+
+	// WarpOffChainMessages encodes off-chain messages (unrelated to on-chain event ie. block or AddressedCall)
```

Comment:
will off-chain block hashes be supported later on? Below it looks like only addressed calls are supported

Reply:
nvm PR descrption says AddressedCall only, but I think worth to add a comment here pointing that out

Reply:
Good call